src/api/notification/documentation/1.0.0/notification.json

```diff
@@ -503,5 +503,169 @@
       ],
       "operationId": "delete/notifications/{id}"
     }
+  },
+  "/notification-list/{account}": {
```

Comment:
I see now why you defined it as `notification-list` you don't want the endpoint to have the same shape as `/notifications/{id}` and `/notifications/{account}`, however, the collection is `notifications` so it feels wrong to introduce an artificial name

What about
`/accounts/{account}/notifications`

Reply:
Renamed